config/parameters/truth/final_approach.txt

```diff
@@ -0,0 +1,26 @@
+# Initial conditions for simulations starting during close approach. These initial
+# conditions were based on output form the following command(s):
+#   python -m psim -s 3250000 -c sensors/base,truth/base,truth/standby --snapshot snapshot.txt OrbitControllerTest
+#
+
+# Time initial conditions.
+
+truth.t.ns 556410000000000
+
+# Leader spacecraft attitude and orbit initial conditions.
+
+truth.leader.attitude.q.body_eci  0.173301   0.971103 0.029768   0.161367
+truth.leader.attitude.w           0.00359105 0.137562 0.00386897
+truth.leader.wheels.w             0.0        0.0      0.0
```

Comment:
It seems like these attitude initial conditions were pulled form a standby case? I think that's the way to go but just wanted to double check!